.pre-commit-config.yaml

```diff
@@ -0,0 +1,36 @@
+default_install_hook_types:
```

Comment:
Would break this commit up in two,

1. Add the hello world driver (including pyproject.py)
2. Add pre-commit configuration